GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6a_with_photoz.yaml

```diff
@@ -0,0 +1,26 @@
+based_on: dc2_object_run2.2i_dr6_with_photoz
+description: DC2 Run 2.2i DR6a Object Catalog with photoz
+catalogs:
+  - catalog_name: dc2_object_run2.2i_dr6
+    matching_method: MATCHING_FORMAT
+    tracts: [2723, 2724, 2725, 2726, 2727, 2728, 2729, 2730, 2731, 2732, 2733, 2734, 2735,
+         2896, 2897, 2898, 2899, 2900, 2902, 2903, 2905, 2906, 2908,
+         3074, 3075, 3076, 3077, 3078, 3079, 3080, 3082, 3083, 3085, 3086,
+         3256, 3257, 3260, 3261, 3262, 3263, 3264, 3265, 3266, 3267, 3268,
+         3441, 3442, 3443, 3444, 3446, 3447, 3448, 3449, 3450, 3451, 3452, 3454,
+         3631, 3633, 3636, 3637, 3638, 3639, 3640, 3641, 3643,
+         3825, 3837,
+         4022, 4035, 4236, 4429, 4440, 4441, 4648, 4860,
+         5071, 5073]
+  - catalog_name: photoz_dc2_object_run2.2i_dr6
+    matching_method: MATCHING_FORMAT
+    tracts: [2723, 2724, 2725, 2726, 2727, 2728, 2729, 2730, 2731, 2732, 2733, 2734, 2735,
+         2896, 2897, 2898, 2899, 2900, 2902, 2903, 2905, 2906, 2908,
+         3074, 3075, 3076, 3077, 3078, 3079, 3080, 3082, 3083, 3085, 3086,
+         3256, 3257, 3260, 3261, 3262, 3263, 3264, 3265, 3266, 3267, 3268,
+         3441, 3442, 3443, 3444, 3446, 3447, 3448, 3449, 3450, 3451, 3452, 3454,
+         3631, 3633, 3636, 3637, 3638, 3639, 3640, 3641, 3643,
+         3825, 3837,
+         4022, 4035, 4236, 4429, 4440, 4441, 4648, 4860,
+         5071, 5073]
+include_in_default_catalog_list: true
```

Comment:
For default list we only list `dr6` (`dr6a` is mainly for people who want to in the future compare `dr6b` and `dr6a`) 
```suggestion
```